src/main/java/io/fabric8/maven/docker/util/WaitUtil.java

```diff
@@ -25,6 +25,8 @@
  */
 public class WaitUtil {
 
+    private static final int DEFAULT_DOCKER_TIMEOUT = 10;
```

Comment:
yeah - looking at your change, i'm not 100% sure that's correct either in terms of returning `wait = 0` b/c you are always waiting some time for the daemon to complete, so i actually think the original implementation was correct.

```
get(long timeout, TimeUnit unit)
Waits if necessary for at most the given time for the computation to complete, and then retrieves its result, if available.```

if anything maybe it should be `if (wait == 0) { wait = DEFAULT }` (i can't remember what i originally had and i'm about to walk out the door for something).